dashboards-observability/public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -73,6 +76,14 @@ export function MissingConfigurationMessage() {
   );
 }
 
+export function processTimeStamp(time: string, mode: TraceAnalyticsMode) {
```

Comment:
could you add some comments for this function? also maybe name should be `standardizeTimeStamp`, but i didn't understand why time is string, but returns a number (jaeger) or string (non-jaeger)?

Reply:
@joshuali925 basically for data-prepper it will be something like now-5y, whereas for jaeger we need to process this to time in microseconds, which is how jaeger stores start time. I will add a comment for it, thanks for pointing out that confusion